### ISSUE TITLE
wttr.in moonphase is just wrong

### ISSUE THREAD

**Issue by narukeh** [LukeSmithxyz/voidrice#636]
The only one who gives the correct one is the package [bsd-games](https://www.archlinux.org/packages/community/x86_64/bsd-games/). I know bsd-games has a lot of stuff some people dont need, so just copy `/usr/bin/pom` to `~/.local/bin/` or `/usr/local/bin/`. `pom` has no deps. (maybe have it ship in larbs?)
The moonphase given from wttr.in (which uses pyphoon) and pom-perl are just wrong. Wrong by 16% percent. (location only differs to max 4%)
Here you can check the correct moonphase: [https://www.timeanddate.com/moon/phases/](https://www.timeanddate.com/moon/phases/) (or any other site)

TLDR: I would suggest to roll back `moonphase` for it to be precise. And suggest the user pom of bsd-games
(in which ever way you decide, close this issue)

**Comment by narukeh** [comment]
Im working on a new script, that gets rid of those problems.....
